### PR TITLE
Update swc_* dependencies

### PR DESCRIPTION
**Pull request by cd-work**

### CODE REVIEW

**Comment by cd-work** [comment]
Seems like a `ListFormat` addition caused the CI to fail. I'd assume the standard procedure would be to just update the expected generation output since it seems like no errors were introduced?

File: rs-lib/src/view/generated.rs [hunk 1]
```diff
@@ -131,6 +131,7 @@ pub enum Node<'a> {
   KeyValuePatProp(&'a KeyValuePatProp<'a>),
   KeyValueProp(&'a KeyValueProp<'a>),
   LabeledStmt(&'a LabeledStmt<'a>),
+  ListFormat(&'a ListFormat<'a>),
```
[the remark this answers is not in the document]
Comment (cd-work) [reply rs-lib/src/view/generated.rs]: Thanks for the tip, just noticed the failure and it would have probably taken me a lot longer to figure this out.

**Review by dsherret — APPROVED** [review]
Thanks! If you [sign the CLA](https://github.com/dprint/dprint-swc-ext/pull/37#issuecomment-1148446019) I can do a release for this.

**Comment by cd-work** [comment]
> Thanks! If you sign the CLA I can do a release for this.

Working on that, will hopefully get that signed by tomorrow.

**Comment by dsherret** [comment]
No, I haven't forgotten it, but I'm planning to do all this either later today or tomorrow. Additionally, I need to do some work pinning swc versions before we upgrade (swc's version bumping is not very good and they often publish broken packages to crates.io so we're going to move away from using swc_ecmascript and instead pin exact versions of each swc crate). Swc upgrades for all our code is usually very involved... last one took a whole week due to breaking changes, but usually they take about a day or half a day of work.

**Pull request merged**